src/common/lambda.rs

```diff
@@ -86,7 +81,7 @@ impl Lambda {
             None => {
                 self.constants.push(data);
                 self.constants.len() - 1
-            },
```

Comment:
I know I probably sounds like a broken record at this point, but another stylistic thing: I prefer trailing commas in match statements, even when used with a block.